app.rb

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env ruby
+
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+class App
+  def initialize
+    @books = []
+    @person = []
+    @rentals = []
+  end
+# rubocop:disable ...
+  def select_opt
+    # rubocop:disable ...
+    option = gets.chomp.to_i
+    case option
+    when 1 then list_books
+    when 2 then list_people
+    when 3 then create_person
+    when 4 then create_book
+    when 5 then create_rental
+    when 6 then list_rentals
+    when 7 then exit(true)
+    else
+      puts 'Invalid number, please try again!'
+    end
+  end
+
+  def list_books
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    @person.each do |pers|
+      puts "[#{pers.class}] Name: #{pers.name}, ID: #{pers.id}, Age: #{pers.age}"
+    end
+  end
+
+  def create_book
+    print 'Title:'
+    title = gets.chomp
+
+    print 'author:'
+    author = gets.chomp
+
+    @books.push(Book.new(title, author))
+
+    puts 'Book created successfully'
+  end
+
+  def check_number(msg)
+    number = 0
+    loop do
+      print msg
+      input = gets.chomp.to_i
+      if input.is_a?(Integer) && input.positive?
+        number = input
+        break
+      else
+        puts 'Please, enter a valid input!'
+      end
+    end
+    number
+  end
+
+  def check_options(msg, options)
+    number = 0
+    loop do
+      print msg
+      input = gets.chomp.to_i
+      if options.include?(input)
+        number = input
+        break
+      else
+        puts 'Please, enter a valid input!'
+      end
+    end
+    number
+  end
+
+  def create_person
+    num = check_options('Do you want to create a student (1) or a teacher (2)? [input the number]: ', [1, 2])
+
+    age = check_number('Age:')
+
+    print 'name:'
+    name = gets.chomp
+    classroom = check_number('Classroom:')
```

Comment:
[REQUIRED] As suggested by the previous reviewer kindy fix the invalid input error after the classroom option while creating a person